### PR TITLE
Request for pyomo.solvers feedstock

### PR DESCRIPTION
**Pull request by whart222**
This is a meta package, which pulls in conda feedstocks that build solvers.

I don't see any practical way to test the feedstock.  Is there a way to verify that solver executables exist?  That would require the execution of a script.

### CODE REVIEW

File: recipes/pyomo.solvers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Do metapackages have a version?
+# Do metapackages have a license?
```
Comment (ocefpaf) [inline recipes/pyomo.solvers/meta.yaml]: Can you remove these comments and the blank lines?

PS: the answer is yes and yes :smile:

File: recipes/pyomo.solvers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+package:
+  name: pyomo.solvers
+  version: 1.0
```
Comment (ocefpaf) [inline recipes/pyomo.solvers/meta.yaml]: Please use a jinja variable for the version.
(You check our example recipe on how to do that.)

File: recipes/pyomo.solvers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
```
[the remark this answers is not in the document]
Comment (whart222) [reply recipes/pyomo.solvers/meta.yaml]: I've found the build section useful to re-build the same meta-package when there are conda-forge build failures.

File: recipes/pyomo.solvers/meta.yaml [hunk 4]
```diff
@@ -1,9 +1,9 @@
-# Do metapackages have a version?
-# Do metapackages have a license?
+# Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (ocefpaf) [inline recipes/pyomo.solvers/meta.yaml]: We discourage comments on recipes unless they are explaining a patch or referencing that needs to be addressed.

**Comment by whart222** [comment]
OK.  I was just copying the comment from the example.

**Comment by ocefpaf** [comment]
> OK. I was just copying the comment from the example.

No problem :smile: but no one read the very first line there:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L1

**Comment by whart222** [comment]
I think this is ready to merge.

**Comment by whart222** [comment]
@ocefpaf Is the merge going to happen soon?

File: recipes/pyomo.solvers/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  commands:
+    - test -f $PREFIX/bin/ipopt  # [unix]
+    - test -f $PREFIX/bin/glpsol  # [unix]
+    - test -f $PREFIX/bin/cbc  # [unix]
+    - if not exist %LIBRARY_BIN%\\ipopt.exe exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\glpsol.exe exit 1  # [win]
+
+about:
+  home: http://pyomo.org
+  license: BSD 3-clause
```
Comment (ocefpaf) [inline recipes/pyomo.solvers/meta.yaml]: We need a license file but that can be done in feedstock later.

**Pull request merged**